### ISSUE TITLE
Exploring ROOMS Warehouse Inventory Coordination with AssetPanda

### ISSUE THREAD

**Issue by caripang** [babybunny/rebuildingtogethercaptain#232]
We would love to know if there was a way to coordinate using AssetPanda to manage warehouse inventory and connecting that with ROOMS orders, etc.  It seems like there is a good foundation with the detail already built into the Items in ROOMS. 

I forwarded the estimate from AssetPanda FYI and attached it:
[Rebuilding Together Peninsula.pdf](https://github.com/babybunny/rebuildingtogethercaptain/files/376586/Rebuilding.Together.Peninsula.pdf)

Assetpanda.com

Included Features

• Unlimited Users

• Level 1 Technical Support

• Barcode Scanning via your Mobile Device

• Cloud-based Storage

• Unlimited Custom Fields and Reporting

• Free Mobile App

• Open API

System Configuration

• Data Upload

• Custom Fields and Groups

• Reports

• User Configuration

• Workflow Implementation

• Web Training

**Comment by babybunny** [comment]
New program and warehouse associate. Good w/ systems. In the office more than previous people. 

"reserve from warehouse" order type. Want to link that to the asset management. 
Should be able to see a list of available assets in that form. 

Make a branch to experiment.  Deploy to a distinct version of the app (not default).  
Add code to see if we can pull something from AssetPanda API.

We'll need to set up a trial of AssetPanda and put in some fake data and see if we can get the API working.  If we can get the systems to talk to each other then we can start thinking about how to enter the data etc.

**Comment by caripang** [comment]
We would still like to explore this option.  Mary Veliz (mary@rebuildingtogetherpeninsula.org) is going to be getting on github eventually and would be a good lead on this from the RTP side to help with organizing/setting this up, with Cari as support.

**Comment by babybunny** [comment]
Yeah, sorry about the delay on this.  I've been focused on getting basic
functionality working in rooms.

**Comment by caripang** [comment]
no worries - that's why i listed it as low priority, Luke.  this is still important upgrade for us to have eventually, but basic functionality for everything else is definitely the priority for ROOMS.
